src/components/transactionsV2/walletTransactionsV2/walletTransactionsV2.js

```diff
@@ -168,6 +175,10 @@ class WalletTransactionsV2 extends React.Component {
 
     const { t, account } = this.props;
 
+    // const delegate = account.isDelegate
```

Comment:
I think you forget to remove this from the code

Reply:
This one is because the delegate tab will be used on the wallet page, but not right now.